Installing-and-Configuring/OS/index.md

```diff
@@ -58,13 +58,19 @@ Your other partitions may be encrypted or not based on your preferences and this
+In this model the TPM enforces rate limits to prevent brute force attacks when entering Disk Unlock Key passphrase, and will prevent further passphrase attempts after 3 bad attempts, requiring to wait or boot the system with Disk Recovery Key passphrase.
 
-On boot, Heads will ask for the TPM disk unlock passphrase that will release the LUKS Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the keys were created.  
+If the disk is moved to a different system the LUKS Disk Recovery Key passphrase in slot 0 may be used to access the encrypted data. This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
 
-This option adds an extra layer of security.  The TPM will only release the LUKS Disk Unlock key if the measurements match.  Generally, an attacker would not have the LUKS Disk Unlock key in keyslot 1 because it was generated by Heads and would not have access to the LUKS disk unlock passphrase in keyslot 0 because it was only used at install.  Even if the attacker clones the drive he will still not have access to the decrypted data.  
+If the Heads TPM drive decryption feature is not disabled, Heads will [ask for](/Keys/#tpm-disk-encryption-key) a Disk Unlock Key passphrase (which might be a PIN: A short passphrase/password) when saving the default boot partition configuration. This is not the same as the passphrase used by LUKS. The TPM Disk Unlock passphrase will be used to generate a LUKS Disk Unlock Key to be placed in LUKS key slot 1 for the root partition. The LUKS Disk Unlock Key is sealed in the TPM along with measurements (LUKS header, firmware measurement, Disk Unlock Key passphrase unlocking TPM NV reserved memory space) taken at that time.
+
+On boot, Heads will ask for the TPM Disk Unlock Key passphrase that will release the LUKS Disk Unlock Key from TPM NV reserved memory. This will only succeed if the current measurements (LUKS header, firmware measurements, Disk Unlock Key passphrase) match the ones taken when the keys were created.
+
+This option adds an extra layer of security. The TPM will only release the LUKS Disk Unlock key if the measurements match (again, LUKS header, firmware measurements and Disk Unlock Key passphrase releasing TPM VN stored Disk Unlocked key if everything is congruent).
+
+Generally, an attacker would not have the LUKS Disk Unlock key in key slot 1, because it was generated by Heads and would not have access to the LUKS Disk Recovery Key passphrase stored under keyslot 0 because it was only used at install, and provided when defining new boot default to nenew/change the Disk Unlock Key passphrase. Even if the attacker clones the drive he will still not have access to the decrypted data, unless he eavesdropped the Disk Recovery Key passphrase.
```

Comment:
Generally, an attacker would not have the LUKS Disk Unlock key in key slot 1, because it was generated under Heads with a random 128 characters key once and then sealed into TPM NV memory space, to be unsealed when provided with proper measurements and Disk Unlock Key passphrase.  To be able to access this content in TPM, or observe that secret traveling from TPM to memory, physical access would be required with the help of a TPM Genie or other physical attack.

The attacker should neither have access to the LUKS Disk Recovery Key nor its passphrase stored under LUKS keyslot 0, because it was setuped at install time, and should only be prompted when defining a new boot default to nenew/change the Disk Unlock Key passphrase after Heads detects changed core /boot components prompts user to detach sign /boot digest (sha256sum). It will then prompt to define a new default boot, which offers to add a disk encryption to the TPM.

Even if the attacker clones the drive, he would still not have access to the decrypted data, unless he eavesdropped the Disk Recovery Key passphrase which was prompted when defining a new boot default, which should be done right after having upgraded QubesOS dom0, from a safe place where eavesdropping is highly improbable.

Reply:
updated.  push in a minute.